$NetBSD$
# HG changeset patch
# User ng0@n0.is
# Date 1561112947 0
#      Fri Jun 21 10:29:07 2019 +0000
# Node ID aaf73ded1091bceebe83aece7c80390c58ebdd63
# Parent  c8b10f57c09660162b34ad143ee294f5ecc5aa0f
main.c: Remove unnecessary variable declaration of errno.

diff -r c8b10f57c096 -r aaf73ded1091 main.c
--- main.c~	Fri Jun 21 10:26:10 2019 +0000
+++ main.c	Fri Jun 21 10:29:07 2019 +0000
@@ -72,7 +72,6 @@
 #endif
 
 /* are these really needed? */
-extern int errno;
 extern int select();
 
 #ifdef USE_REGEXP
